x/superfluid/keeper/hooks_test.go

```diff
@@ -67,8 +89,8 @@ func (suite *KeeperTestSuite) TestSuperfluidAfterEpochEnd() {
 				suite.Require().True(found)
 				suite.Require().Equal(sdk.NewDec(7500000), delegation.Shares)
 			}
-			balance := suite.App.BankKeeper.GetAllBalances(suite.Ctx, delAddrs[0])
-			suite.Require().Equal(tc.expRewards, balance)
+			//balance := suite.App.BankKeeper.GetAllBalances(suite.Ctx, delAddrs[0])
+			//suite.Require().Equal(tc.expRewards, balance)
```

Comment:
Is this intentional? Isn't this the check for expected rewards at the end?

Reply:
We shouldn't have commented out code in general :P